data-sources/aws-chatbot/config.yml

```
@@ -0,0 +1,8 @@
+id: aws-chatbot
+displayName: aws chatbot
```

Sorry one more thing, since this is a displayName lets give proper casing to the product name

```suggestion
displayName: AWS Chatbot
```

same with the description below "Monitor AWS Chatbot access"

After that we should be good to go!

---

@MichelLosier, addressed, Thank You For Helping, Going Forward i will keep all points in mind.